### DISCUSSION TITLE
Scheduled Top Up?

### DISCUSSION THREAD

**Discussion by kerberjg** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#513] [Q&A]
Hi, I've been considering buying the Pro version, but I'm wondering, can the Top Up be scheduled?

For example, I leave every morning to go to my office at 8AM, Mon-Fri, and I'd like the battery to be charged to 100% by then.

After that however, I'd rather use the 80% limit or Sailing Mode, as my laptop is plugged in at the office.

Is this currently supported somehow?

**Comment by MatthiasKerbl** [comment]
Hi @kerberjg,

Unfortunately, right now we do not have any scheduling feature implemented. Therefore, you would need to activate Top Up manually.

Cheers,
Matthias
